### PR TITLE
ci: continue if peer deps job fails

### PR DESCRIPTION
**Pull request by barmac**
This aims to allow us to discover quickly whether a CI failure is caused by incompatibility or another problem.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -2,25 +2,25 @@ name: CI
...
+        integration: [ true ]
         integration-deps:
         - diagram-js@11.9 bpmn-js@11.5
         - "@bpmn-io/properties-panel@3"
-        - "" # as defined in package.json
-
-    runs-on: ${{ matrix.os }}
+        include:
+          - integration-deps: "" # as defined in package.json
+            integration: false
```
Comment (nikku) [inline .github/workflows/CI.yml:15]: What does this do? I don't see `integration` being used.

Comment (barmac) [reply .github/workflows/CI.yml:15]: `integration` is a notion that a job is part of the `integration-deps` suite. This is used to determine whether other jobs should continue if the job fails via `continue-on-error`: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/987/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R6

Comment (barmac) [reply .github/workflows/CI.yml:15]: Alternatively, we could call this property `continue-on-error` but I didn't want to mix up GHA keywords and our custom properties. If it makes more sense, I will rename this.

**Comment by barmac** [comment]
BTW with this settings we can see clearly that the latest release of `@bpmn-io/properties-panel` causes the failure.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -2,25 +2,25 @@ name: CI
 on: [ push, pull_request ]
 jobs:
   Build:
-
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.integration }}
```
Comment (nikku) [inline .github/workflows/CI.yml:6]: This is the trick, thanks!

**Review by nikku — APPROVED** [review]
Makes sense. Cool stuff.

**Pull request merged**